### ISSUE TITLE
Minecraft Region Fixer doesn't work on my Windows 10 version. It crashes.

### ISSUE THREAD

**Issue by TheIntelloBox** [Fenixin/Minecraft-Region-Fixer#57]
Hi,

Minecraft Region Fixer doesn't work on my Windows 10 version. It crashes.

**Comment by ryantheleach** [comment]
It's designed for Java Edition, and to my knowledge the last update to this repo was older then the windows 10 world format even existing.

**Comment by Fenixin** [comment]
That's right. Minecraft Region Fixer is designed for java edition (or the first minecraft).

Right now there is no plan for adding support to windows 10 version, at least for my part.  Maybe I should add some information about windows 10 version/java version.

**Comment by TheIntelloBox** [comment]
Yes please add some information because it doesn't work. :)

**Comment by TheIntelloBox** [comment]
Please could you add support for Windows 10, my map doesn't work anymore...

**Comment by Fenixin** [comment]
Well, adding support for windows 10 minecraft is not an easy task so I'm afraid it's not going to happen soon. Maybe after some time, when my free time is what it used to be.

But right now is not going to happen.

**Comment by TheIntelloBox** [comment]
Yes the GUI doesn't work on Windows 10, it crashes. :c

**Comment by Fenixin** [comment]
Ah, that's quite different from adding support to windows 10 minecraft edition. I will probably look into it in christmas.

**Comment by TheIntelloBox** [comment]
Yeah I just wanted Windows 10 support, not Minecraft Windows 10 support. xD

**Comment by DracoDynasty** [comment]
It crashed for me too on Windows 10.

Managed to run it without problems using Windows 7 compatibility mode, though.

**Comment by Fenixin** [comment]
Yeah, sorry about that. Right now I don't have the time to fix it, so please, try using the command line interface.

**Comment by msmcleish** [comment]
Fenixin is there any specifications I could get a hold of for the composition off the files in the db folder?

My son and I have been working on a world together for several years (created with mcpe) and it has corrupted chunks and this is devastating to my son.

Any advice?  Links?  Anything?  Anybody please help.

Regards,
Marshall.

**Comment by Fenixin** [comment]
Hello @msmcleish!

I'm sorry to say that I have no information about Minecraft pocket edition and that Region-Fixer doesn't work on MCPE worlds. If I were you I would start by looking in the official wiki, I think this is a relevant link: 

https://minecraft.gamepedia.com/Bedrock_Edition_level_format

I would also try to search for world editors and tools, maybe you cas restore some parts of the world using them, see these links:

https://minecraft.gamepedia.com/Programs_and_editors
https://minecraft.gamepedia.com/Programs_and_editors/Mapping#Map_Editors

I hope that helps. If you find anything, could you please report with your findings? So it's useful for others. Don't hesitate to ask if you need anything else.

**Comment by msmcleish** [comment]
Thank you!  I will dig into these links and see if I can find some
solution.  I will be sure to report on  any relevant result.

Thanks again,
Marshall.

**Comment by macfreek** [comment]
@msmcleish Have you had any luck in fixing your corrupt world? If not, why don't you create a new bug report, and we'll see what we can do to help you. You now only made a comment in an unrelated older ticket. Please add the string "@macfreek" somewhere in your new ticket to alert me.

The first two steps you should do is make a backup of your current (corrupt) world folder. You are going to edit it, and there is a good chance you make things worse. Secondly, check if you have a backup from the world while it was still good.

Since I suspect you don't have a recent good backup, your next two steps are to detect what part of your world is correct, and then replace just that part.

I maintained the underlying library that Minecraft-Region-Fixer uses to read and write the NBT files. Perhaps I can guide you a bit reading those files.

For example, [https://github.com/twoolie/NBT/tree/master/examples there is a script there] called `regionfile_analysis.py` which you can use to see if there is something wrong with your files.

**Comment by msmcleish** [comment]
Thank you for taking the time to get back to me.  I will create a ticket
this afternoon.

In the meantime, does mcpe/windows 10 versions of Minecraft use MBT files?
I thought it was a different file format I was dealing with (nnnnnn.dat)
etc.

Marshall.

P.S
I will create a ticket and I will be sure to include the tag and my
question.about the MBT files for the.benefit of the.community.
